dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/QueryLogSharedStateTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.edwmigration.dumper.application.dumper.QueryLogSharedState.QueryLogEntry;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class QueryLogSharedStateTest {
+
+  @Before
+  public void beforeEachTest() {
+    QueryLogSharedState.clearQueryLogEntries();
+  }
+
+  @Test
+  public void queryLogFirstEntryUpdatedSuccessfully() {
+    // Arrange
```

Comment:
The `// Arrange` comment can be omitted in tests, because it can be assumed that the first section of the test will be `Arrange`.